### ISSUE TITLE
Unable to upload file to JED

### ISSUE THREAD

**Issue by mgilkes** [joomla/jed-issues#235]
Hi. I was just trying to update my extension to the latest release on my JED listing. When trying to upload the new release file, I get an error message saying "Failed to move uploaded file". Check the screenshot. Thanks.
<img width="1381" alt="JED-upload-failed" src="https://user-images.githubusercontent.com/6433533/143725945-79b21ec0-4ea3-478a-84ff-943ab66d0581.png">

**Comment by Llewellynvdm** [comment]
Hi @mgilkes thanks for reporting this, we moved the JED to new hardware Yesterday. We will during this coming week iron out any issue like these. Will keep you posted.

**Comment by Llewellynvdm** [comment]
Can you please try again... as this seems to be resolved now. Can you confirm this?

**Comment by mgilkes** [comment]
Hi @Llewellynvdm. I just tried to upload again, but I get the same error. Thanks for following up. I hope you can sort it out soon. The file I am uploading is about 2.7MB.
<img width="1241" alt="Screen Shot 2021-12-03 at 9 04 33 AM" src="https://user-images.githubusercontent.com/6433533/144607670-ab543790-8ffb-4759-8e36-91ae3b20a721.png">

**Comment by Llewellynvdm** [comment]
Okay our team has been testing this, and after a week we are still not seeing this error. Can you try another browser, and or computer? Or just try clearing you browser cache?

Apologies for any inconvenience, but I would really like to get to the bottom of this, and make sure it gets resolved some how.

**Comment by mgilkes** [comment]
Hi. OK, I just tried it in Chrome, but I decided to try it with two different zip files. One is the UNZIPFIRST zip file, that contains the editions of my extension for Joomla 3.10 and 4.0. The other is only the zip file that relates to Joomla 4.0. The size difference between the files is that the first is 2.7MB and the second is 552KB. (The 4.0 edition is smaller since I removed some image files from my updated extension). The result was that I got the  same error with the UNZIPFIRST zip file, and it worked without any error for the smaller 4.0 edition zip file. Is there a file size upload limitation with your uploader?

I didn't have this problem just the day before I opened this issue. At that time, I successfully uploaded the previous version of my UNZIPFIRST zip file. It was also 2.7MB.
<img width="1450" alt="Screen Shot 2021-12-05 at 12 18 11 AM" src="https://user-images.githubusercontent.com/6433533/144733861-22df629a-8dec-4086-b48a-422563cdbba5.png">
<img width="1447" alt="Screen Shot 2021-12-05 at 12 34 39 AM" src="https://user-images.githubusercontent.com/6433533/144733874-f92193dd-e9dd-46e1-85d6-ed7aaf820309.png">
